util/cache-osm

```diff
@@ -26,7 +26,7 @@ if [[ $URL == "null" ]]; then
     exit 1
 fi
 
-curl $URL > /tmp/$CC.osm.pbf
+wget $URL -O /tmp/$CC.osm.pbf
```

Comment:
`wget` isn't installed by default in a lot of distros, any particular reason to change this from curl?

Reply:
you are right, I got some issues with curl and my internet, and  I notice that  `wget` download faster than curl , that is why I have changed, and I forgot setting up in the docker container.